Guides/YAML Quickstart.md

```diff
@@ -0,0 +1,248 @@
+
+## Information needed to create YAML files
+
+1. The model that is used for training.
+2. Some basic knowledge about MAX78000 hardware.
+
+Following the instructions below, you will be able to create a layer-by-layer description of your model in the YAML format. Once completed, copy the YAML file to ai8x-synthesis/networks and reference it in the synthesis script for your trained model.
+
+ 
+
+## YAML File Structure
+
+The YAML file includes global configuration in the header, followed by a layer-by-lauerdescription which correspond to the layers in the model.
+
+### 1. Global Configuration Section
+
+Using the “`arch`“ and “`dataset`“ keywords, the global section must include the name of the model and dataset as used for training. These keywords are used to make sure a YAML file matches a trained model, and to pull in a sample data file for the known-answer test (KAT) that is auto-generated by the synthesis tool.
+
+Example: MNIST
+
+*./train.py --epochs 200 --deterministic --compress schedule.yaml --model **ai85net5** --dataset **MNIST** --confusion --param-hist --pr-curves --embedding --device MAX78000*
+
+```yaml
+# ------------   Global Configuration   ------------
+arch: ai85net5  # Mandatory: use the name of the model as in training
+dataset: MNIST  # Mandatory: use the name of the dataset as in training 
+```
+
+
+
+### **2. Layer Description Section**
+
+The layer descriptions section defines each layer as it appears in the model. This section starts with the “`layers`“ keyword, followed by the configuration of each layer.  
+
+Use “`-`” as the delimiter to start each layer. The order of keywords in the description of a layer is arbitrary. 
+
+```yaml
+# ------------   Layers   ------------
+- pad: 0
+  activate: ReLU
+  out_offset: 0x2000
+  processors: 0xffffffffffffffff
+  data_format: HWC
+  operation: Conv1d
+  kernel_size: 1
+```
+
+
+
+The following figure shows how a layer in the model is mapped to YAML:
+
+![YAML mapping](resources/yaml-mapping.png)
+
+
+
+The table below shows the description of frequently used keywords. For a complete list of keywords and more description, please check the main documentation [README.md](https://raw.githubusercontent.com/MaximIntegratedAI/MaximAI_Documentation/develop/README.md).
+
+| keyword      | Description                                                  | Available options                                            |
+| ------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| data_format  | **ONLY in first layer**, specifies the organization of data in memory | CHW or HWC                                                   |
```

Comment:
Maybe we can add the information of maximum data size for HWC and state something like if your 2d data is less than 90x91 per channel, you can use use HWC as default. This is because CHW can create the issue you mentioned in common pitfalls. HWC is more noob friendly :)

Reply:
Added HWC recommendation